docs/scenarios/parametric.md

```diff
@@ -144,10 +144,13 @@ TEST_LIBRARY=python PYTHON_DDTRACE_PACKAGE=git+https://github.com/Datadog/dd-tra
 
 #### NodeJS
 
-There is two ways for running the NodeJS tests with a custom tracer:
+There is three ways for running the NodeJS tests with a custom tracer:
 1. Create a file `nodejs-load-from-npm` in `binaries/`, the content will be installed by `npm install`. Content example:
     * `DataDog/dd-trace-js#master`
 2. Clone the dd-trace-js repo inside `binaries`
+3. Create a file `nodejs-load-from-local` in `binaries/`, this will disable installing with `npm install dd-trace` and
+   will instead mount the `../dd-trace-js` directory in the container and then `npm link` to it. This also removes the
```

Comment:
A universal way is to get the content of `nodejs-load-from-local`, and use it to have the location of the repo. 

```suggestion
   will instead get the content of the folder, and use it as a location of the `dd-trace-js` repo and then `npm link` to it. For instance, if this repo is at the location, you can set the content of this file to `../dd-trace-js`. This also removes the
```

Reply:
obviously have `../dd-trace-js` be the default value as i think most devs use it too

Reply:
@simon-id I just used what's in the file without a default since the file needs to be created anyway. If you really think it would make it easier to use to have a default I can add it as well.

Reply:
actually nvm you're right